scripts/generate-app-config/generatePagesConfig.js

```diff
@@ -228,6 +326,11 @@ const generatePagesConfig = (siteConfig, production, verbose) => {
   // Build out the page config from the discovered file paths.
   const config = buildPageConfig(filePaths, generatePagesOptions, siteConfig.npmPackage.name);
 
+  // Check config here
+  if (appConfig.includeTestEvidence) {
+    config.evidence = buildTestEvidenceConfig(generatePagesOptions, siteConfig.npmPackage.name);
```

Comment:
Given the complexity of creating the test evidence config, I feel like this should get pushed to its own file to make this easier to read.
We could generate Test Evidence Config [here](https://github.com/cerner/terra-dev-site/pull/109/files#diff-f443b80960106092c69c1e822a17a01fR45) and then possibly merge the Pages and Test Evidence configs before we pass to the [generate routes config](https://github.com/cerner/terra-dev-site/pull/109/files#diff-f443b80960106092c69c1e822a17a01fR65